interfaces/builtin/network_manager.go

```diff
@@ -103,6 +103,10 @@ network packet,
 
 #include <abstractions/nameservice>
 
+# Explicitly deny ptrace. This doesn't influence any NetworkManager
+# functionality but silences AppArmor denials in the system log.
+deny ptrace (trace) peer=###PLUG_SECURITY_TAGS###,
```

Comment:
It would be nice if this comment was more specific. Ie:

```
# Explicitly deny plugging snaps from ptracing the slot to silence noisy
# denials. Neither the NetworkManager service nor nmcli require ptrace
# trace for full functionality.